From 7a56432a48f6af4b2e5c2f0ec62ff974b111a3e9 Mon Sep 17 00:00:00 2001
From: Stephan Mueller <smueller@chronox.de>
Date: Wed, 27 Jul 2022 07:21:45 +0200
Subject: [PATCH] set default log level to NONE

Libraries should not issue any logs. Thus, set the default log level to
NONE.

Suggested-by: Fabian Vogt
Signed-off-by: Stephan Mueller <smueller@chronox.de>
---
 lib/kcapi-kernel-if.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/kcapi-kernel-if.c b/lib/kcapi-kernel-if.c
index 0df1a02..d7b10bf 100644
--- a/lib/kcapi-kernel-if.c
+++ b/lib/kcapi-kernel-if.c
@@ -55,7 +55,7 @@ void orig_memset_secure(void *s, int c, uint32_t n)
 /************************************************************
  * Logging logic
  ************************************************************/
-enum kcapi_verbosity kcapi_verbosity_level = KCAPI_LOG_ERR;
+enum kcapi_verbosity kcapi_verbosity_level = KCAPI_LOG_NONE;
 
 void kcapi_dolog(enum kcapi_verbosity severity, const char *fmt, ...)
 {
